src/components/DepositCrypto/index.tsx

```diff
@@ -69,7 +78,10 @@ const DepositCrypto: React.FunctionComponent<DepositCryptoProps> = (props: Depos
                     <p className="cr-deposit-info">{text}</p>
                     {data ? <div className="d-none d-md-block qr-code-wrapper"><QRCode dimensions={size} data={data}/></div> : null}
                 </div>
-                <div>
+                <div className="cr-deposit-crypto__block">
+                    {currency === 'eth' && !isMobileDevice ? (
+                        <MetaMaskButton depositAddress={data} />
+                    ) : null}
```

Comment:
can you write this in one line or move to the separate method

Reply:
why? There is no need for additional method here

Reply:
The reason is show or hide this button
Better to write in either in one line or move this the separate function